qrhl/qrhl.el

```diff
@@ -116,8 +129,9 @@
   proof-save-command-regexp "\\`a\\`"   ;AKA `regexp-unmatchable' in Emacs-27
   proof-tree-external-display nil
   proof-script-font-lock-keywords qrhl-font-lock-keywords
-  proof-goals-font-lock-keywords qrhl-font-lock-keywords
+  proof-goals-font-lock-keywords qrhl-font-lock-subsuperscript
```

Comment:
Does this mean you print goals with a smaller font? Interesting.

Reply:
@Matafou, no, `qrhl-font-lock-subsuperscript` is for one very specific purpose. Isabelle/HOL (which is used internally in qrhl-tool) uses `⇩` and `⇧` to make the next letter a sub- or superscript. We cannot convert this into Unicode sub/superscripts because Unicode has only a very limited set of sub/superscripts. (Actually, we do use Unicode sub/superscripts in the few cases where they exist.) So I wrote this font-lock config that replaces `⇩x` visually by a subscript `x` etc. (It's actually used not only in the goals, `qrhl-font-lock-keywords` includes `qrhl-font-lock-subsuperscript`.)

Reply:
> (It's actually used not only in the goals, qrhl-font-lock-keywords includes qrhl-font-lock-subsuperscript.

Why would the two buffers have different fonts then?

Reply:
It's not a different font. It's a different font-lock configuration for the two buffers. The script buffer has `qrhl-font-lock-keywords` which highlights keywords *and* does the sub/superscript stuff. The goal buffer has `qrhl-font-lock-subsuperscript` which only does the sub/superscript stuff but does not highlight keywords (because keywords are specific to the script language.)

Reply:
Ho I see. Thanks. In Coq we usually highlight the goals buffer, because  highlight terms and types, not only commands.